src/itx.rs

```diff
@@ -0,0 +1,141 @@
+use crate::include::common::bitdepth::AsPrimitive;
+use crate::include::common::bitdepth::BitDepth;
+use crate::include::common::bitdepth::FromPrimitive;
+use crate::include::common::bitdepth::ToPrimitive;
+use crate::include::common::intops::iclip;
+use crate::include::common::intops::imin;
+use crate::include::stddef::*;
+use crate::include::stdint::*;
+
+extern "C" {
+    fn memset(_: *mut libc::c_void, _: libc::c_int, _: libc::c_ulong) -> *mut libc::c_void;
+}
+pub type itx_1d_fn =
+    Option<unsafe extern "C" fn(*mut int32_t, ptrdiff_t, libc::c_int, libc::c_int) -> ()>;
+
+pub unsafe extern "C" fn inv_txfm_add_rust<BD: BitDepth>(
```

Comment:
This only deduplicates this function in the module, right?  Could you name the commit and PR something like "`fn inv_txfm_add_rust`: Deduplicate with generics" for the next ones?

Reply:
Will do, thanks.